aws/resource_aws_api_gateway_rest_api_test.go

```diff
@@ -406,7 +408,7 @@ func TestAccAWSAPIGatewayRestApi_api_key_source(t *testing.T) {
 	resourceName := "aws_api_gateway_rest_api.test"
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t); testAccAPIGatewayTypeEDGEPreCheck(t) },
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
Keep existing `PreCheck` function:

```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccAPIGatewayTypeEDGEPreCheck(t) },
```